diagnostic.md

```diff
@@ -184,5 +215,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+25 px.
```

Comment:
At 550px, the `@media(min-width= 400px)` takes over and so `.stretch` is 25%. So 25% * 550px = 137.5px